docs/install_with_helm.md

```diff
@@ -22,7 +22,7 @@ wget https://github.com/IBM/ansible-lifecycle-driver/releases/download/<version>
 Install the chart using the helm CLI:
 
 ```
-helm install ansiblelifecycledriver-<version>.tgz --name ansible-lifecycle-driver [--set app.config.override.lifecycle.request_queue.topic.num_partitions=100,app.config.override.lifecycle.request_queue.topic.replication_factor=1]
+helm install ansible-lifecycle-driver ansiblelifecycledriver-<version>.tgz [--set app.config.override.lifecycle.request_queue.topic.num_partitions=100,app.config.override.lifecycle.request_queue.topic.replication_factor=1]
```

Comment:
Looks good. Let's clarify above that this is to install the chart using Helm v3 CLI

Reply:
@dvaccarosenna yes, this install command is for Helm v3 CLI. I have tested it in OCP. Before running this install command I have checked the Helm version and then executed the Helm Install command. Please find the below snapshot.

![image](https://user-images.githubusercontent.com/106679486/184806772-871db20d-4842-4455-acf6-ae2da4f176ae.png)

Reply:
Sorry @suryadipnag, what I mean is let's mention that in the docs, so a user knows that Helm v3 is required.

Reply:
ok @dvaccarosenna , Now I have added the version in the doc.